README.md

```diff
@@ -225,3 +189,33 @@ curl -s http://0.0.0.0:8080/block | jq
 ## Contribute
 
 Need help or want to contribute ideas or code? Head over to our [CONTRIBUTING](CONTRIBUTING.md) doc for more information.
+
+## Note for maintainers
+
+All the commits in this repo follow the Conventional Commits spec. When merging a PR, make sure 1/ to
+use squash merge and 2/ that the title of the PR follows the Conventional Commits spec. This will
+make squash the PR into one conventional commit.
+
+The history of commits will be used to generate the `CHANGELOG`. To do so, run yarn deploy on the master
```

Comment:
I really like https://keepachangelog.com/en/1.0.0/ and try to promote using it everywhere. Not sure how close that is to what is generated though.

Reply:
What is generated looks like this: https://github.com/paritytech/txwrapper/blob/master/CHANGELOG.md

I think it is very close to that except there is no unreleased section.